client/rest-high-level/src/main/java/org/elasticsearch/client/TasksClient.java

```diff
@@ -61,4 +63,37 @@ public void listAsync(ListTasksRequest request, ActionListener<ListTasksResponse
         restHighLevelClient.performRequestAsyncAndParseEntity(request, RequestConverters::listTasks, ListTasksResponse::fromXContent,
                 listener, emptySet(), headers);
     }
+
+    /**
+     * Cancel one or more cluster tasks using the Task Management API
+     * <p>
+     * See
+     * <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/tasks.html"> Task Management API on elastic.co</a>
+     * </p>
+     */
+    public CancelTasksResponse cancelTasks(CancelTasksRequest cancelTasksRequest, Header... headers) throws IOException {
```

Comment:
can we call the methods just cancel given that they now belong to the TasksClient?